indexers/vector/MilvusIndexer/milvusdbhandler.py

```diff
@@ -98,11 +98,11 @@ def insert(self, keys: 'np.ndarray', vectors: 'np.ndarray'):
             db.insert(reduce(operator.concat, keys.tolist()), vectors)
 
     def build_index(self, index_type: str, index_params: dict):
-        t = self.get_index_type(index_type)
+        type = self.get_index_type(index_type)
```

Comment:
remove changes with Milvus because it cannot be updated and at least there will be a mapping between last published image and actual code